docs/UserGuide.adoc

```diff
@@ -448,13 +509,34 @@ Marks the session with the unique session id of 25 as done.
 
 ==== Filter `filter`
 
-TODO: ADD DETAILS ONCE IMPLEMENTED
+Filters undone sessions fulfilling all provided constraints.
+
+All undone sessions with matching keywords are displayed. Keyword is case-insensitive.
```

Comment:
you should specify that the keywords can only be mod code, session type or date. Also, what happens if the user just enters session filter with no other keywords?